src/main/java/com/capitalone/dashboard/event/TestResultEventListener.java

```diff
@@ -154,7 +155,7 @@ private void extractPerformanceMetrics(TestCase testCase, TestCapability testCap
                     targetErrorRate = Double.valueOf(testCaseStep.getDescription());
                 }
                 if (testCaseStep.getId().equalsIgnoreCase(STR_ACTUAL_ERROR_RATE)){
-                    double actualErrorRate = Double.valueOf(testCaseStep.getDescription());
+                    actualErrorRate = Double.valueOf(testCaseStep.getDescription());
```

Comment:
What if the `testCaseStep.getDescription()` is empty? Are there checks to prevent the `java.lang.NumberFormatException`?

Reply:
@Sbrenthughes Checks added.